components/title/multisize/src/index.js

```diff
@@ -0,0 +1,79 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+// Map object for title sizes and HTML Tags.
+const TAG_FOR_SIZE = {
+  'l': 'h1',
+  'm': 'h2',
+  's': 'h3'
+}
+
+/**
+ * Builds an HTML/React Element depending on the size declared (s: H3, m: H2, l: H1).
+ * @param size {string} Size (s, m, l) of title element.
+ * @param className {string} CSS classNames to apply to element.
+ * @param content {element} Children of title element. Could be an element (HTML/React Element), string, or an array of both.
+ * @returns {element} React Title Element.
+ * @private
+ */
+const _buildTitleElement = (size, className, content) => {
```

Comment:
Removed! I had not seen it, I guess it was autogenerated by the WebStorm's jsDoc when it detected the underscore dangle in this function name.